lib/aPI.ml

```diff
@@ -22,9 +8,79 @@ include Rpc
 type string_list = string list [@@deriving rpc]
 
 module Ref = struct
-  type 'a t=string
-  let of_string x = x
+  (* BEGIN ref.ml from xen-api repository *)
```

Comment:
I think if we modified this after copying it from `xen-api` then we could mention it here, cause otherwise this comment can be a bit confusing:
`Modifications: replaced Stdext.Xstringext with Astring`